src/App.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-import { connect } from 'react-redux';
-
-import NavigationContainer from './nav/NavigationContainer';
-
-const App = (props) =>
-  <NavigationContainer />;
-
-const mapStateToProps = (state) => ({
-  isLoggedIn: state.app.isLoggedIn,
```

Comment:
That is for a case when the apiKey has been changed, I guess we are not handling it right now, but is needed.